src/App.jsx

```diff
@@ -12,6 +13,7 @@ function App() {
             <Route path="/" element={<LandingPage />} />
             <Route path="/login" element={<LoginPage />} />
             <Route path="/roadmap" element={<RoadmapPage />} />
+            <Route path="/taskcard" element={<TaskCard />} />
```

Comment:
I've created a PR that caters to this, so the route path here can be modified to `<Route path="/:task" element={<TaskCard />} />`
where `task` can now be used to determine what task is shown (e.g the linkedin page, resume, etc)